package.json

```diff
@@ -4,11 +4,12 @@
   "description": "Ubiquibot plugin template repository with TypeScript support.",
   "author": "Ubiquity DAO",
   "license": "MIT",
+  "main": "src/worker.ts",
   "engines": {
     "node": ">=20.10.0"
   },
   "scripts": {
-    "format": "run-s format:lint format:prettier format:cspell",
```

Comment:
I think `run-s` is preferred over `yarn &&` see [here](https://github.com/ubiquity/rpc-handler/pull/28#discussion_r1642103541) although I'm not sure of the difference between the two

Reply:
I see. My motivation behind removing it is that this package has not been maintained over 6 years. We could replace it with [concurrently](https://github.com/open-cli-tools/concurrently) which is more recent and better maintained, achieving the same functionalities.

Reply:
Would be better as a separate issue to replace across the org if taking this approach. I'm okay with `yarn &&`

Reply:
`run-s` means run in series. `run-p` means run in parallel.